environments/ptl/ptl.tfvars

```diff
@@ -148,3 +152,27 @@ additional_routes = [
 
 additional_routes_coreinfra = [
 ]
+
+################################################################################
+# Kubernetes configuration
```

Comment:
Could we catchup on this format on Slack? 
- we want to keep the current component/variable setup, rather than combining
- No need for ###, variables should be user readable. Comments add unnecessary bloat